.circleci/config.yml

```diff
@@ -27,6 +27,9 @@ jobs:
             cd website 
             yarn install --frozen-lockfile
             yarn build
+      - run:
+          name: Ping FeaturePeek
+          command: bash <(curl -s https://peek.run/ci)
```

Comment:
Would probably be best to only run this on non-master branches, since the master branch is going to be deployed to the live site.

Reply:
@joeldenning I've added a line to ignore the master branch on the `build-website` job – please double-check that that's correct.